tests/test_other.py

```diff
@@ -61,6 +61,19 @@ def decorated(self):
   return decorated
 
 
+def check_python3_version():
+  """emscripten requires at least python3.5 since python3.4 and below do not
+  support circular dependencies."""
+  python3 = Building.which('python3')
+  if not python3:
+    return False
+  output = run_process([python3, '--version'], stdout=PIPE).stdout
+  version = [int(x) for x in output.split(' ')[1].split('.')]
+  if version < [3, 5, 0]:
```

Comment:
`return version >= [3, 5, 0]`?
Also didn't know that comparison worked on lists, that's really elegant.